docs/Onboarding/Prerequisites_Guide.md

```diff
@@ -0,0 +1,90 @@
+- A Github organization that can be used for testing within OpenShift CI.
+- Layered product deployment automation in public GitHub repositories within that org.
+- Layered product testing automation in public GitHub repositories repo within that org.
+### 2. Private GitHub repositories
+- A Github organization that can be used for testing within OpenShift CI.
+- Layered product deployment automation in private GitHub repositories within that org (mirrored from your downstream source).
+- Layered product testing automation in private GitHub repositories repo within that org (mirrored from your downstream source).
+> **IMPORTANT:**
+>
+>If you need to use private Github repos it is the product QE's responsibility to create the Github org, mirror the downstream repos to Github private repos, and maintain that mirror for the lifetime of the scenario.
+
+## GitHub Org Requirements
+- [Add Openshift-ci-robot & openshift-merge-robot](https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#granting-robots-privileges-and-installing-the-github-app) to your Github organization.
+
+## GitHub User Readiness
+- Add github usernames to the openshift organization ([follow this guide](https://source.redhat.com/groups/public/atomicopenshift/atomicopenshift_wiki/openshift_onboarding_checklist_for_github)).
+
+## Containerized Product Deployment
+- The product can be installed via the operator hub (for GA'd layered products).
+- A method to install the operator from the operator hub within a container image.
+  - If your product is able to be installed using the steps existing in the  [operatorhub-subscribe ref](https://github.com/openshift/release/tree/master/ci-operator/step-registry/operatorhub/subscribe) than we can make use of that instead.
+- Product install configuration (anything that needs to be done post operator install) executed within a container (maintained by the product QE team in upstream repo or reachable registry).
+- Identify all environment variables and secrets needed for install.
+
+See the [Container Creation Guide](../OCP_CI_Tutorials/Containers/Container_Creation_Guide.md) for general guidance when creating your layered product deployment image(s).
+
+## Containerized Tests
+- Product test environment setup executed within a container image (maintained by product QE team in test repo or reachable registry).
+- A script or command that can be run against the test container to trigger and generate an XML for the tests.
+- Identified all environment variables and secrets needed for testing.
+
+See the [Container Creation Guide](../OCP_CI_Tutorials/Containers/Container_Creation_Guide.md) for general guidance when creating your scenarios test image(s).
+
+## Resolve Internal Dependencies
+- Confirmation that the containerized deployment and testing can be run outside of the company firewall and doesn't rely on internal dependencies.
+
+## Valid Test Output
+- Produce J-unit XMLs in a valid format for each test that is executed.
```

Comment:
```suggestion
- Produce JUnit XMLs in a valid format for each test that is executed.
```

Reply:
fixing in next commit

Reply:
Thanks for pointing this out Caleb. My understanding is that we should be focusing on the JUnit XML schema. For what reason I'm not 100% sure on.
- It seems to have a broader set of tools that integrate/support it, maybe that's why.

I can do a search replace for Xunit and update this everyone in this PR if you agree.

Reply:
That sounds good to me as long as we are consistent in the documentation!